packages/editor/src/lib/config/createTLStore.ts

```diff
@@ -48,7 +44,9 @@ export type TLStoreSchemaOptions =
 	  }
 
 /** @public */
-export type TLStoreOptions = TLStoreBaseOptions & { id?: string } & TLStoreSchemaOptions
+export type TLStoreOptions = TLStoreBaseOptions & { id?: string } & TLStoreSchemaOptions &
+	/** Collaboration options for the store. */
+	Pick<TLStoreProps, 'collaboration'>
 
```

Comment:
let's leave this specified in the base options, it results in much better reference docs than picking it off this way